### PR TITLE
Positioning fixes for search

### PR DESCRIPTION
**Pull request by jimmynotjim**
Positioning fixes to stop Search trigger and form from stacking while transitioning

## Changes

- Adds absolute positioning to search form and a set width in the organism

## Removals

- Removes z-index from search button (form should sit on top of the button).

## Testing

- `gulp build` and test the menu in the header

## Review

- @anselmbradford 
- @KimberlyMunoz 
- @sebworks 

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: cfgov/unprocessed/css/organisms/header.less [hunk 1]
```diff
@@ -83,6 +83,10 @@
             & > .m-global-search {
                 float: right;
             }
+
+            > .m-global-search {
+                width: 340px;
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/header.less]: @anselmbradford I got this from the existing form width, it's necessary to use absolute positioning on the form.

Comment (anselmbradford) [inline cfgov/unprocessed/css/organisms/header.less]: Should this be min-width?

Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/header.less]: Yep, not sure why I didn't go that route. Updated.

**Comment by KimberlyMunoz** [comment]
:+1:

**Comment by sebworks** [comment]
:+1:

**Comment by anselmbradford** [comment]
:+1:

**Pull request merged**